ArcGISRuntimeSDKQt_CppSamples/Layers/QueryOGCAPICQLFilters/README.metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "category": "Layers",
+    "description": "Query data from an OGC API feature service using CQL filters.",
+    "ignore": false,
+    "images": [
+        "screenshot.png"
```

Comment:
screenshot needs to be updated with the QtQuick Controls 2 update. Also, screenshots should be 500x500 pixels, and we generally try and take a platform agnostic screenshot (don't include the windows or mac top bar)